command/tls.go

```diff
@@ -0,0 +1,58 @@
+package command
+
+import (
+	"os"
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type TLSCommand struct {
+	Meta
+}
+
+func fileDoesNotExist(file string) bool {
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		return true
+	}
+	return false
+}
+
+func (c *TLSCommand) Help() string {
+	helpText := `
+Usage: nomad tls <subcommand> <subcommand> [options]
+
+This command groups subcommands for creating certificates for Nomad TLS configuration. 
+The TLS command allows operators to generate self signed certificates to use
```

Comment:
Just a quibble around wording here. These aren't "self-signed certificates" in the traditional meaning of that term. They're signed by a CA, it's just that we're _also_ creating the CA.